front/components/DataSourceViewResourceSelectorTree.tsx

```diff
@@ -95,6 +100,13 @@ function DataSourceViewResourceSelectorChildren({
     parentId,
     viewType,
   });
+  const [showConnectorPermissionsModal, setShowConnectorPermissionsModal] =
+    useState(false);
+
+  const { connector } = useConnector({
```

Comment:
This should not be here! Not all data source view has a connector, this will lead to a lot of 401s on many nodes. Can we please remove it from here?

Reply:
Omw to the office, fixing once there